index.html

```diff
@@ -155,9 +177,7 @@ <h2 class="recentwork">
             <li><a href="#">Javascript</a></li>
             <li><a href="#">html</a></li>
           </ul>
-          <button class="resumebutton buttons" type="submit">
-            See Project
-          </button>
+          <button class="resumebutton buttons" type="submit">See Project</button>
```

Comment:
- [ ] Instead of using `margin-left` to center the project buttons kindly use `flexbox` to align the contents of the cards as mentioned in the requirements, it's also better to use flexbox because it's responsive and the button will always be centered no matter the size of the container 👍 

![image](https://user-images.githubusercontent.com/50721479/206475147-15cbc6a8-1b05-443d-a251-08dd829eafdb.png)